src/modules/users/crons/profile-populator.cron.ts

```
@@ -8,50 +9,97 @@ import ProfilePopulator from 'src/common/helper/profileUpdate/profile-update';
+
+  @Cron('*/1 * * * *')
   async populateProfile() {
     try {
       console.log("Profile Populator CRON started " + new Date());
-      const users = await this.getUsers();
+      const users = await this.getUsersV2();
       await this.profilePopulator.populateProfile(users);
     } catch (error) {
       Logger.error("Error in 'Profile Populator CRON': ", error);
       throw new Error("Error in 'Profile Populator CRON': " + error);
+      throw new Error("Error in 'Profile Populator CRON': " + error);
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)